core/pubnub_coreapi.c

```diff
@@ -7,6 +7,8 @@
 #include "pubnub_assert.h"
 #include "pubnub_timers.h"
 #include "pubnub_memory_block.h"
+// TODO: why not all pbcc?
```

Comment:
is TODO still applicable?

Reply:
Yeah. It was my question but I think that's C sourcing stuff and I think that I don't want to create a discussion about it. I will remove that todo.